src/js/components/Calendar/Calendar.js

```diff
@@ -589,23 +590,36 @@ const Calendar = forwardRef(
       return (
         <Box direction="row" justify="between" align="center">
           <Box flex pad={{ horizontal: headingPadMap[size] || 'small' }}>
-            <Heading
-              level={
-                size === 'small'
-                  ? (theme.calendar.heading && theme.calendar.heading.level) ||
-                    4
-                  : ((theme.calendar.heading && theme.calendar.heading.level) ||
-                      4) - 1
-              }
-              size={size}
-              margin="none"
-              overflowWrap="normal"
-            >
-              {reference.toLocaleDateString(locale, {
-                month: 'long',
-                year: 'numeric',
-              })}
-            </Heading>
+            {theme.calendar.heading && theme.calendar[size].titleSize ? (
+              <Text weight="bold" size={theme.calendar[size].titleSize}>
```

Comment:
Enforcing bold here might not be the best idea because in v5 of HPE theme, we are leaning away from use of bold in typography. Wondering if we should have `theme.calendar[size].title` where any text props are supported?

In base.js, the default weight might be bold but then in HPE theme we can override with 500 or whatever is appropriate?

Reply:
good idea!

Reply:
> Wondering if we should have theme.calendar[size].title where any text props are supported?

Done in latest commit